index.html

```diff
@@ -35,11 +36,166 @@ <h1 class="my-name">Hey There, <br> I'm Gbenga</h1>
                     <li><a href="#"><img src="./images/angelistic.png" alt="angelistic"></a></li>
                     <li><a href="#"><img src="./images/twitter.png" alt="twitter"></a></li>
                     <li><a href="#"><img src="./images/medium.png" alt="medium"></a></li>
-
                 </ul>
 
             </div>
         </section>
     </div>
+    <section class="works">
+        <h2>My Recent Works</h2>
+        <hr>
+        <img src="./images/recent-works.png" alt="work-card">
+        <div class="posts-container">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <div class="works-btn">
+                <button type="button">css</button>
+                <button type="button">html</button>
+                <button type="button">bootstrap</button>
+                <button type="button">Ruby</button>
+            </div>
+            <button  class=" see-works cards-btn"  type="button">See Project</button>
+        </div>
+        <br>
+        <br>
+          <div class="works-cards">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="works-card-btn">
+                <button type="button">html</button>
+                <button type="button">bootstrap</button>
+                <button type="button">Ruby</button>
+          </div>
+          <button type="button" class="cards-btn">See Project</button>
+        </div>
+    
+        <div class="works-cards">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+                standard</p>
+            <div class="works-card-btn">
+                <button type="button">html</button>
+                <button type="button">bootstrap</button>
+                <button type="button">Ruby</button>
+            </div>
```

Comment:
- You are required to enclose the buttons that list technologies in a `ul` tag 👍 . Please work on this